html-forms/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="#" method="#" id="coding-form">
+      <div>
+        <label for="email">Email</label>
+      </div>
+      <input type="text" name="email" id="email">
+
+      <div>
+        <label for="why">Why is coding cool?</label>
+      </div>
+      <select name="why">
+        <option value="">Select an option</option>
+        <option value="Because it's interesting">Because it's interesting</option>
+        <option value="Because it's fun">Because it's fun</option>
+        <option value="Because it's powerful">Because it's powerful</option>
+      </select>
+
+      <div>
+        <label for="language"><p>Favorite Language (choose one)</p></label>
+      </div>
+      <div>
+        <input type="radio" name="language" value="HTML"> HTML
+        <input type="radio" name="language" value="CSS">CSS
+        <input type="radio" name="language" value="JavaScript">JavaScript
+      </div>
+
+      <div>
+        <label for sign-up p"><p>Would you like to sign up for more information?</p></label>
```

Comment:
need to fix this `for` attribute

Reply:
![image](https://user-images.githubusercontent.com/97073517/151487444-77753ec1-70d1-4698-98d4-825b04ef6883.png)